bot/core.py

```
@@ -932,15 +936,57 @@ def init(
+    auth_mode_normalized = str(auth_mode or "api_key").strip().lower() or "api_key"
+
+    # Clear any stale ccproxy-injected env vars BEFORE resolve_provider()
+    # reads them — otherwise a prior OAuth session's ANTHROPIC_BASE_URL /
+    # sentinel key would silently pin the next API-key mode request to the
+    # local ccproxy endpoint (Bug 3: OAuth pollutes API-key mode).
+    try:
+        from omicsclaw.core.ccproxy_manager import clear_ccproxy_env
+        clear_ccproxy_env()
+    except Exception:
+        # ccproxy_manager is optional; missing module just means no cleanup needed.
+        pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)